deploy/cr.yaml

```diff
@@ -97,3 +97,16 @@ spec:
 
     serverHost: monitoring-service
     serverUser: admin
+
+  backup:
+    image: perconalab/percona-server-mysql-operator:xtrabackup
```

Comment:
I think we need to use official docker image `percona/percona-xtrabackup` .

Reply:
We'll be able to use official image after https://github.com/percona/percona-docker/pull/599 merged.

For restores I'll see what we can do but it's good to go for backups.